packages/babel-plugin-flow-runtime/src/createConversionContext.js

```diff
@@ -7,6 +7,8 @@ export type Options = {
   assert?: boolean;
   warn?: boolean;
   annotate?: boolean;
+  suppressComments?: RegExp[];
```

Comment:
options often get passed in from `.babelrc` which is (often) JSON, so we'll need to accept strings here and convert to RegExps